src/System.IO/tests/Stream/Stream.NopStream.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public sealed class NopStream : Stream
```

Comment:
@stephentoub There is existing code that uses this class [here](https://github.com/dotnet/corefx/blob/master/src/System.IO/tests/Stream/Stream.TimeoutTests.cs#L54) I simply moved it to a new file. Should I move it back and delete the lines using it? (It looks like the tests in that file are already using `MemoryStream`, BTW; they just added an additional class for the sake of further testing.)